### DISCUSSION TITLE
Allow indivual page note even when there are already site-wide notes present

### DISCUSSION THREAD

**Discussion by egabrum** [manugarg/pagenotes-issues#71] [Ideas]
If I have a note for www.abc.com , I currently cannot add specific notes for www.abc.com/page1 and www.abc.com/page2

**Comment by manugarg** [comment]
You can. If you uncheck "Apply to all pages of this site.", while saving notes for `www.abc.com/page1`, it will be page specific note. I've many notes like this myself.
